pkg/skaffold/deploy/helm/helm.go

```diff
@@ -303,7 +303,7 @@ func (h *Deployer) Deploy(ctx context.Context, out io.Writer, builds []graph.Art
 	for ns := range nsMap {
 		namespaces = append(namespaces, ns)
 	}
-	deployedImages, _ := manifests.GetImages(manifest.NewResourceSelectorImages(manifest.TransformAllowlist, manifest.TransformDenylist))
+	deployedImages, _ := manifests.GetImages(manifest.NewResourceSelectorImages(h.transformableAllowlist, h.transformableDenylist))
```

Comment:
Hey @beastpu. This part looks good. Could you please also update https://github.com/GoogleContainerTools/skaffold/blob/1f591bafe367868e61eedc38f427acaae61e0a31/pkg/skaffold/deploy/helm/helm.go#L627 to use `h.transformableAllowlist` and `h.transformableDenylist`? We'll need to send those to the `warnAboutUnusedImages` function; it will remove a warning when using the Helm deployer with a CRD. Thanks a lot 😄